.eslintrc.js

```diff
@@ -77,6 +78,39 @@ module.exports = {
     'unicorn/no-array-push-push': 'error',
     'unicorn/prefer-keyboard-event-key': 'error',
     'unicorn/prefer-regexp-test': 'error',
-    'unicorn/prefer-string-replace-all': 'error'
+    'unicorn/prefer-string-replace-all': 'error',
+    '@intlify/vue-i18n/no-dynamic-keys': 'error',
+    // TODO: enable at a later date. currently disabled to prevent massive conflicts for initial PR
+    // '@intlify/vue-i18n/no-unused-keys': [
+    //   'error',
+    //   {
+    //     extensions: ['.js', '.vue', 'yaml']
+    //   }
+    // ],
+    '@intlify/vue-i18n/no-duplicate-keys-in-locale': 'error',
+    '@intlify/vue-i18n/no-raw-text': [
+      'error',
+      {
+        attributes: {
+          '/.+/': [
+            'title',
+            'aria-label',
+            'aria-placeholder',
+            'aria-roledescription',
+            'aria-valuetext',
+            'tooltip',
+            'message'
+          ],
+          input: ['placeholder', 'value'],
+          img: ['alt']
+        },
+        ignoreText: ['-', '•', '/', 'YouTube', 'Invidious', 'FreeTube']
+      }
+    ],
+  },
+  settings: {
+    'vue-i18n': {
+      localeDir: './static/locales/*.yaml'
```

Comment:
Could you please only include the files for the locales that are allowed to be used in FreeTube, that should hopefully be slightly faster and will prevent potential linting issues from cropping up for files that we don't use.

You could do it in the same way as we do for the WatchExternalFiles plugin in `webpack.renderer.config.js`.